app/Models/WaitingRoom.php

```diff
@@ -15,4 +15,62 @@ public function users()
     {
         return $this->belongsToMany(User::class);
     }
+
+    /*
+     * Equally splits the users of a waiting room into arrays
+     * representing competitions.
+     */
+    public function getDefaultSplit()
+    {
+        $users = $this->users;
+
+        // Get the size of the waiting room
+        $roomSize = count($users);
+
+        // Determine the amount of competitions to make
+        $numOfCompetitions = $roomSize / 300;
```

Comment:
:+1:  i had this fear early on when doing the math on paper but in practice it didnt prove to be a problem, so perfectly valid concern!